python/servo/package_commands.py

```diff
@@ -52,6 +53,10 @@
     'mac': [
         'production/servo-tech-demo.dmg',
     ],
+    'openharmony': [
+        'openharmony/aarch64-unknown-linux-ohos/production/'
```

Comment:
Does this line mean the shared library will be archived? If yes, that would be great, since it would allow also simply downloading and packaging a prebuilt servo library from the external ServoDemo repository.

Reply:
Nope, this is just using python's string syntax to break a single long line. I've now removed this line as we don't have a step in the workflow to upload the packages to github releases. Adding new published packages should probably happen in a separate PR and maybe we can consider prebuilt `.so` on GH release as part of that.